common/topology/src/serde.rs

```diff
@@ -181,7 +192,7 @@ pub struct SerializableGateway {
     // optional ip address in the case of host being a hostname that can't be resolved
     // (thank you wasm)
     #[cfg_attr(feature = "wasm-serde-types", tsify(optional))]
-    #[serde(alias = "explicit_ip")]
+    #[serde(alias = "explicit_ips")]
```

Comment:
Is this correct?

Reply:
I changed that to match the changes I did on that variable name earlier. 
It wasn't used by the localnet script, so it wasn't creating any problems. I don't know where else it's used though.